docs/alma/alma.rst

```diff
@@ -294,6 +294,58 @@ their types.
       velocity_resolution  double          m/s        Estimated velocity resolution from all the spectral windows, from frequency resolution.
 
 
+Query Results
+=============
+
+Results of queries are returned in tabular format. For convenience,
+the `~astroquery.alma.get_enhanced_table` function can be used to have the initial result
+in a more useful format, i.e., turn values into quantities, footprint into
+shape, etc. (Note: this require the `regions` Python package to be installed.
+
+
+.. doctest-remote-data::
+.. plot::
```

Comment:
Oh, I'm sorry, I didn't mention this before. The `plot` directive restarts the context, so it needs to be a completely standalone code snippet. 

However, as with all the directives, empty lines and inserted narratives ends it, thus you run into the failures.